.github/workflows/ci-test.yml

```diff
@@ -141,5 +141,7 @@ jobs:
           ./bootstrap.sh -DENABLE_GC=ON -DCMAKE_BUILD_TYPE=RELEASE -DENABLE_UNIFIED_COMPILATION=ON && cd build && make -j2
 
     - name: Run tests (MacOS)
+      # TODO: Renable gtest. Unit tests currently fail on MacOS.
```

Comment:
@fruffy Would you be able to open a GitHub issue describing the problem and add a link in the comment, or we could add a link to this pull request?